### ISSUE TITLE
Training dataset dice score lower than validition dataset

### ISSUE THREAD

**Issue by Shawn0099** [woodywff/brats_2019#6]
hi,
I trained your network for about 100 epoch. My training loss is lower than validition loss,but dice_coefficient for training dataset is lower than vadlition dataset.I also drew boxplot using evaluate.py,and my validition result is almost same as the result in your paper,but the training dataset result is worse.Is there anything wrong with my settings?
Another question is about calculating hausdorff distence.I've calculated sensitivity and specicity and my result is close to yours,but hausdorff distence is much higher than normal(about 20).I used SimpleITK.HausdorffDistanceImageFilter.Could you tell me how you do this?
Many thanks!

**Comment by woodywff** [comment]
The second answer comes first :-) I didn't calculate the four metrics (dice, hausdorff, sensitivity and specificity) by myself. [CBICA's evaluation system](https://ipp.cbica.upenn.edu/) would take care of that. You just need to Sign up, Log in, and upload your prediction results, then you'll get the metrics feedback.  
`brats_2019/demo_task1/draw_evaluation.py` will plot the figs based on the downloaded `.csv` files.

When you said training and validation dataset, how many subjects have you taken into account?
The final training is on the whole 335 subjects, and the validation is for the 125 subjects.
Also notice that I mentioned two patching strategies in the article and for each of them the network has been trained for 100 epochs.
That's what came to my mind right now.  Good luck to you! Keep in touch :-)
